receiver/sqlserverreceiver/README.md

```diff
@@ -47,6 +47,14 @@ Top-Query collection specific options (only useful when top-query collection are
   - Queries that were finished execution outside the lookback window are not included in the collection. Increasing the lookback window (in seconds) will be useful for capturing long-running queries.
 - `max_query_sample_count` (optional, example = `5000`, default = `1000`): The maximum number of records to fetch in a single run.
 - `top_query_count`: (optional, example = `100`, default = `200`): The maximum number of active queries to report (to the next consumer) in a single run.
+- `collection_interval`: (optional, default = `60s`): The interval at which top queries should be emitted by this receiver.
+  - This value must be a string readable by Golang's [time.ParseDuration](https://pkg.go.dev/time#ParseDuration).
```

Comment:
This line is unneeded information, a user does not care the underlying type and parsing.

Reply:
Updated